packages/static-module-record/test/fixtures/exportheavy.js

```diff
@@ -42,9 +41,6 @@ export {
 // eslint-disable-next-line import/export
 export * from './src/types.js';
 
-
-const { details: X, Fail } = assert;
```

Comment:
This seems like dead code, so I deleted it as a drive by. Nothing to do with the purpose of this PR